### PR TITLE
Bump secapr to 1.1.0

### PR DESCRIPTION
**Pull request by druvus**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/secapr/meta.yaml [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
-  sha256: ada58295f1d02ca2da0b57a49d74770afcb3ac3137c25c32ecf19bb44937dbfa
+  fn: secapr_{{ version }}.tar.gz
+  url: https://github.com/AntonelliLab/seqcap_processor/archive/v{{ version }}.tar.gz
+  sha256: 988a37eaf9fc050f482c0ae1acb9994ce144a04d2de9ae1b827469398e7bb5dd
 
 build:
   skip: True  # [not py27]
   number: 0
-  script: $PYTHON setup.py install
+  script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (mbargull) [inline recipes/secapr/meta.yaml]:
```yaml
  entry_points:
    - secapr = secapr.__main__:main
```
to use a Conda-created entry point.

Comment (druvus) [reply recipes/secapr/meta.yaml]: thanks @mbargull! Adding the entry point info.

**Comment by druvus** [comment]
Trying to resolve unknown issues in #6887.

File: recipes/secapr/meta.yaml [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
+  entry_points:
+    - secapr = secapr.__main__:main
 
 requirements:
   build:
     - python
+    - setuptools
     - biopython
     - pandas
```
Comment (mbargull) [inline recipes/secapr/meta.yaml]:
```diff
-    - biopython
-    - pandas
```
`setup.py` is nice and doesn't import anything apart from `sys`, `setuptools` and `versioneer` 😄

File: recipes/secapr/meta.yaml [hunk 3]
```diff
@@ -1,27 +1,28 @@
...
+
   run:
     - python
     - biopython
```
Comment (mbargull) [inline recipes/secapr/meta.yaml]: `numpy` should also be a run requirement.

Comment (druvus) [reply recipes/secapr/meta.yaml]: good catch! thanks!

Comment (tobiashofmann88) [reply recipes/secapr/meta.yaml]: thanks for figuring this out and sorry for the complications!

**Pull request merged**